lib/pages/character/inventory.dart

```diff
@@ -90,61 +90,77 @@ class InventoryPage extends StatelessWidget {
-                    child: Text(
-                      bag.itemInfo.name,
-                      style: Theme.of(context).textTheme.bodyText1.copyWith(
-                        fontWeight: FontWeight.w500
-                      )
+          Container(
+            padding: EdgeInsets.all(8.0),
+            child: Row(
+              children: <Widget>[
+                if (bag.itemInfo != null)
+                  CompanionItemBox(
+                    item: bag.itemInfo,
+                    hero: '$bagIndex ${bag.id}',
+                    size: 45.0,
+                    includeMargin: false,
+                  ),
+                if (bag.itemInfo != null)
+                  Expanded(
+                    child: Padding(
+                      padding: const EdgeInsets.all(8.0),
+                      child: Text(
+                          bag.itemInfo.name,
+                          style: Theme
+                              .of(context)
+                              .textTheme
+                              .bodyText1
+                              .copyWith(
+                            fontWeight: FontWeight.w500,
+                            fontStyle: fontStyle
+                          )
+                      ),
                     ),
                   ),
-                ),
-              if (bag.itemInfo == null)
-                Spacer(),
-              Text(
-                '$usedSlots/${bag.size}',
-                style: Theme.of(context).textTheme.bodyText1,
-              )
-            ],
+                if (bag.itemInfo == null)
+                  Spacer(),
+                Text(
+                  '$usedSlots/${bag.size}',
+                  style: Theme.of(context).textTheme.bodyText1.copyWith(
+                    fontStyle: fontStyle
+                  ),
+                )
+              ],
+            ),
           ),
+          if (usedSlots > 0)
+            Divider(),
           if (usedSlots > 0)
             Container(
               width: double.infinity,
-              margin: EdgeInsets.only(top: 8.0),
+              margin: EdgeInsets.only(top: 8.0, bottom: 8.0),
+              alignment: AlignmentDirectional.center,
               child: Wrap(
                 spacing: 4.0,
                 runSpacing: 4.0,
                 children: inventory
-                  .where((i) => i.id != -1)
-                  .map((i) => CompanionItemBox(
-                    item: i.itemInfo,
-                    skin: i.skinInfo,
-                    hero: '$bagIndex ${inventory.indexOf(i)} ${i.id}',
-                    upgradesInfo: i.upgradesInfo,
-                    infusionsInfo: i.infusionsInfo,
-                    quantity: i.charges != null ? i.charges : i.count,
-                    includeMargin: false,
-                  ))
-                  .toList(),
+                    .where((i) => i.id != -1)
+                    .map((i) =>
+                    CompanionItemBox(
```

Comment:
I'd revert this part, as I personally think it looked better previously.

Reply:
This isn't one of my changes?